testscommon/unsignedTxHandlerMock.go

```diff
@@ -64,7 +64,7 @@ func (ut *UnsignedTxHandlerMock) CleanProcessedUTxs() {
 }
 
 // ProcessTransactionFee -
-func (ut *UnsignedTxHandlerMock) ProcessTransactionFee(cost *big.Int, devFee *big.Int, txHash []byte) {
+func (ut *UnsignedTxHandlerMock) ProcessTransactionFee(cost *big.Int, _ *big.Int, txHash []byte) {
```

Comment:
include `devFee` in `ProcessTransactionFeeCalled` as to be consistent with the functions' signatures.
Also, you can rename this struct to stub instead of mock

Reply:
fixed